src/index.html

```diff
@@ -7,6 +7,40 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar">
```

Comment:
```suggestion
    <div class="calendar calendar--month... calendar--.....">
```
<img width="829" alt="Screenshot 2023-12-08 at 16 18 11" src="https://github.com/mate-academy/layout_calendar/assets/135232001/5b6a7296-8f56-4746-a8b3-39aaf07e1e6e">

Reply:
Can u please reply for my questions in MA chats ? I tag u.